CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # BuildingSync
 
+## Version 2.2.0-pr1
+
+BuildingSync Version 2.2.0-pr1 does not introduce any breaking changes. It introduces elements required for 
+a complete audit report for ASHRAE Standard 211 Level 1.  
+Note that the element `auc:PrimaryHVACSystemType` is in the process of being deprecated, and users should replace it with 
+`auc:PrincipalHVACSystemType` to ensure future conformance to the schema.
+
+Date Range: 07/31/20 - 08/31/20
+
+| Category       | Count |
+|----------------|-------|
+| Controls         | 0  |
+| General         | 2  |
+| Measures         | 0  |
+| Reports         | 7  |
+| Systems         | 1  |
+| Validation         | 0  |
+| Other         | 2  |
+| **Total**      | 12 |
+
+| Change Type    | Count |
+|----------------|-------|
+| Breaking Change         | 0  |
+| Non-breaking Change         | 0  |
```

Comment:
hmm. looks like the non-breaking change didn't work... should be 12, right?

Reply:
Oops, forgot to add those tags